frontend/public/locales/en/apply-adult.json

```
@@ -32,7 +37,8 @@
       "date-of-birth-month-required": "Date of birth must include a month",
       "date-of-birth-valid": "Day must be valid for the given month and year",
       "date-of-birth-year-number": "Year must be a number, for example 1950",
-      "date-of-birth-year-required": "Date of birth must include a year"
+      "date-of-birth-year-required": "Date of birth must include a year",
+      "dtc-required": "Select whether or not you have a valid Disability Tax Credit certificate"
```

Nitpick: maybe use current prod's error message?

```suggestion
      "dtc-required": "Select whether or not you have a valid disability tax credit"
```